truenas/webui
# NAS-113451 / 22.02-RC.2 / ix-forms: Dashboard Configuration

To test:

- Make sure the dashboard configuration form saves selections to backend and dash reflects changes saved from the form.
- Deselect all the widgets in the form and make sure the empty state action button pulls up the correct form
- Make sure Overview widgets (Storage/Network) always come before widgets with identifiers (Pool/Interface)
